docs/examples/alibi-detect/.gitignore

```
@@ -0,0 +1 @@
+alibi-detector-artifacts
```

Maybe do this
```suggestion
alibi-detector-artifacts/*
```

and keep the folder so that whoever's running the notebook doesn't need to create the folder and can run it out of the box

---

Might need to create the directory with an empty `.keep` file in it so git actually controls it if we want the folder to be available out-of-the-box.

---

Yeah on this one I can add the empty folder but just checking since I think other runtimes artifacts are also in git @adriangonz ?

---

I think that the artifacts for some examples are there actually (not for tests though).

---

okay added empty folder now